cmd/commands/namespace.go

```diff
@@ -92,3 +97,40 @@ func NamespaceInit(manifests map[string]*core.Manifest, kc *core.KubectlClient)
 
 	return command
 }
+
+func NamespaceCleanup(kc *core.KubectlClient) *cobra.Command {
+	options := core.NamespaceCleanupOptions{}
+
+	command := &cobra.Command{
+		Use:   "cleanup",
+		Short: "cleans up " + env.Cli.Name + " resources in the namespace",
+		Long: fmt.Sprintf(`cleans up %s resources in the namespace, 
+including the secret if one is specified with "--secret" and
+the namespace itself if "--remove-ns" is set`, env.Cli.Name),
+		Example: fmt.Sprintf(`  %[1]s namespace cleanup default
+  %[1]s namespace cleanup default --remove-ns
+  %[1]s namespace cleanup default --secret push-credentials
+  %[1]s namespace cleanup default --remove-ns --secret push-credentials`, env.Cli.Name),
+		Args: ArgValidationConjunction(
+			cobra.ExactArgs(namespaceCleanupNumberOfArgs),
+			AtPosition(namespaceCleanupNameIndex, ValidName()),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			options.NamespaceName = args[namespaceCleanupNameIndex]
+
+			if err := (*kc).NamespaceCleanup(options); err != nil {
+				return err
+			}
+
+			PrintSuccessfulCompletion(cmd)
+			return nil
+		},
+	}
+
+	LabelArgs(command, "NAMESPACE")
+
+	command.Flags().BoolVarP(&options.RemoveNamespace, "remove-ns", "", false, "removes the namespace as well")
```

Comment:
What's the point? If you're going to delete the namespace just run `kubectl delete ns <name>` and call it a day.

`riff namespace init` doesn't create the namespace, it feels asymmetrical for riff to delete the namespace.

Reply:
Ah, I missed that we do that.